streams/src/main/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImpl.java

```diff
@@ -109,69 +75,34 @@ private void determineIsQueryable(final String queryableStoreName) {
         Objects.requireNonNull(reducer, "reducer can't be null");
         Objects.requireNonNull(materialized, "materialized can't be null");
         final MaterializedInternal<K, V, KeyValueStore<Bytes, byte[]>> materializedInternal
-                = new MaterializedInternal<>(materialized, builder, REDUCE_NAME);
+                = new MaterializedInternal<>(materialized, builder, AGGREGATE_NAME);
```

Comment:
> `we mistakenly used REDUCE_NAME for the prefix`

Why "mistakenly" -- it's a `reduce(...)` operator? Because `count()` is an special case of `aggregation()` it seems to be fine to use `AGGREGATE_NAME` there?

Reply:
Sounds good, will leave a note on the upgrade path as well.